trunk-recorder/call.cc

```diff
@@ -179,6 +183,29 @@ void Call::end_call() {
       myfile.close();
     }
 
+    if (sys->get_daily_log()) {
+      std::ofstream myfile2(dailylog_filename, std::ofstream::app);
```

Comment:
I know in the end it doesn't matter, and that the status json files are also called in as `myfile`, but for readability would you be willing to rename both that and `myfile2` to something more descriptive while you're here?

Reply:
doing